2996_reflection/reflection.md

```diff
@@ -3944,8 +4164,8 @@ Adjust paragraph 3 of [temp.arg.general] to not apply to splice template argumen
     f<int()>();       // int() is a type-id: call the first f()
 
 +   constexpr int x = 42;
-+   f<[:^int:]>();    // splice-template-argument: calls the first f()
-+   f<[:^x:]>();      // splice-template-argument: calls the second f()
++   f<[:^int:]>();    // splice-specifier: calls the first f()
++   f<[:^x:]>();      // splice-specifier: calls the second f()
```

Comment:
Can we just change this example to add `#1` and `#2` to the two fs so that we can say calls `#1`, calls `#2`? I think we usually do that.

Reply:
Done.